jOOQ-codegen/src/main/java/org/jooq/codegen/JavaGenerator.java

```diff
@@ -459,7 +459,8 @@ public final void generate(Database db) {
         log.info("  table-valued functions", generateTableValuedFunctions());
         log.info("  tables", generateTables()
               + ((!generateTables && generateRecords) ? " (forced to true because of <records/>)" :
-                ((!generateTables && generateDaos) ? " (forced to true because of <daos/>)" : "")));
+                ((!generateTables && generateDaos) ? " (forced to true because of <daos/>)" :
+                ((!generateTables && generateIndexes) ? " (forced to true because of <indexes/>)" : ""))));
```

Comment:
(I looked but I couldn't find any tests to update to ensure the new behavior works correctly, I presume they are elsewhere. Nonetheless, should be pretty much a no-brainer anyway.)

Reply:
Thanks for your PR. The tests are not open source, see https://github.com/jOOQ/jOOQ/blob/main/CONTRIBUTING.md

Reply:
Thanks. :+1: I suspected this could be the case in fact. I read the whole CONTRIBUTING guide now, thanks for the link - it'll help me balance between "fix myself" and "only file an issue" moving forward.

Reply:
> it'll help me balance between "fix myself" and "only file an issue" moving forward.

Yeah, non-trivial fixes are probably not practical given the lack of access to tests... Also, a lot of things aren't as trivial as they may first seem.

Reply:
Yep, true that. :+1: Btw, I noticed (after reading the fine CONTRIBUTING guide) that I should have added my own name to `jOOQ/src/main/resources/META-INF/ABOUT.txt` as well. :see_no_evil: Feel free to fix it if you like.

Reply:
> Feel free to fix it if you like.

It's optional, only if you want to

Reply:
Yes, I wouldn't mind. :+1: https://github.com/jOOQ/jOOQ/pull/13012